runtime/js/99_main.js

```diff
@@ -89,6 +93,61 @@ ObjectDefineProperties(Symbol, {
 let windowIsClosing = false;
 let globalThis_;
 
+const ALREADY_WARNED_DEPRECATED = new SafeSet();
+
+function warnOnDeprecatedApi(apiName, stack) {
+  if (ALREADY_WARNED_DEPRECATED.has(apiName + stack)) {
+    return;
+  }
+
+  // If we haven't warned yet, let's do some processing of the stack trace
+  // to make it more useful.
+  const stackLines = StringPrototypeSplit(stack, "\n");
+  ArrayPrototypeShift(stackLines);
+  while (true) {
+    // Filter out internal frames at the top of the stack - they are not useful
+    // to the user.
+    if (stackLines[0].includes("(ext:") || stackLines[0].includes("(node:")) {
+      stackLines.shift();
+    } else {
+      break;
+    }
+  }
+  // Now remove the last frame if it's coming from "ext:core" - this is most likely
+  // event loop tick or promise handler calling a user function - again not
+  // useful to the user.
+  if (stackLines[stackLines.length - 1].includes("(ext:core/")) {
+    stackLines.pop();
+  }
+
+  ALREADY_WARNED_DEPRECATED.add(apiName + stack);
+  console.log(
+    "%cWarning",
+    "color: yellow; font-weight: bold;",
+  );
+  console.log(
```

Comment:
When importing from `deno_std` without a pinned version (e.g. `import * as http from "https://deno.land/std/http/server.ts`), the warning printed has a yellow "Warning" prefix and unformatted rest-of-text. Should this warning be consistent with that?
![image](https://github.com/denoland/deno/assets/29347852/066f6411-60ce-4298-b88f-06c013c9ae8c)

Reply:
No, I want it to stand out and be sore to the eyes :)

Reply:
Good idea 😆